docs/MO_DG/prepare_model/Model_Optimizer_FAQ.md

```diff
@@ -586,56 +586,56 @@ For Tensorflow:
 
 For all frameworks:
 1. [Replace cycle containing Sub-graph in Model Optimizer](customize_model_optimizer/Customize_Model_Optimizer.md)
-2. [OpenVINO™ Extensibility Mechanism](../../Extensibility_UG/Intro.md)
+2. [OpenVINO&trade; Extensibility Mechanism](../../Extensibility_UG/Intro.md)
 
 or
 * Edit model in original framework to exclude cycle.
 
-#### 98. What does the message "Can not transpose attribute '..' with value .. for node '..' .." mean?  <a name="question-98"></a>
+#### 94. What does the message "Can not transpose attribute '..' with value .. for node '..' .." mean?  <a name="question-94"></a>
 
 This message means that model is not supported. It may be caused by using shapes larger than 4-D.
 There are two ways to avoid such message:
 
-1. [Cutting Off Parts of a Model](convert_model/Cutting_Model.md)
+1. [Cut Off Parts of a Model](convert_model/Cutting_Model.md).
 2. Edit network in original framework to exclude such layers.
 
-#### 99. What does the message "Expected token `</ParallelComponent>`, has `...`" mean?  <a name="question-99"></a>
+#### 95. What does the message "Expected token `</ParallelComponent>`, has `...`" mean?  <a name="question-95"></a>
 
-This error messages mean that Model Optimizer does not support your Kaldi model, because the Net contains `ParallelComponent` that does not end by tag `</ParallelComponent>`.
+This error messages mean that Model Optimizer does not support your Kaldi model, because the Net contains `ParallelComponent` that does not end with `</ParallelComponent>` tag.
```

Comment:
```suggestion
This error messages mean that Model Optimizer does not support your Kaldi model, because the Net contains `ParallelComponent` that does not end with the `</ParallelComponent>` tag.
```